tests_proc_dict.py

```diff
@@ -175,3 +218,44 @@ def test_participant_id_column_goes_through(participant_annotation, user_dict):
     assert result.get("participant_id").get("Annotations").get("Identifies") is not None
 
 
+def test_nan_is_read_as_string(missing_file):
+    result = load_annotations(missing_file)
+    assert result.isPartOf[0] == ""
+    assert result.value[0] == "n/a"
+    assert result.type[0] == "object"
+
+
+def test_unusual_strings_are_identical_in_output(unusual_file, tmp_path):
```

Comment:
Is the purpose of this test just an extra check to ensure that values are read in as strings? Or also a check that the manual annotation processing is 'lenient' enough to allow annotations of seemingly unmeaningful values to pass?

Reply:
I'm not quite sure what you mean. The purpose of the string is to check that if a value has been mapped to `nb:MissingValue` in the input, that it ends up in the `MissingValues` section of the annotations, and not in the `Levels`. I see that the test is not written in a way that makes this clear. I'll refactor it

Reply:
Oh, I thought that that was the purpose of `test_missing_value_is_parsed_correctly()`. The test I'm referring to here (i.e., `test_unusual_strings_are_identical_in_output()`) seems to be using a test tsv which has values like `"nan"` / `"True"` / `"n/a"` annotated as `nb:Male`.

Reply:
Oh. Hold on. I think I tricked myself here :). I removed something further up and your comment showed up on a different test.

Reply:
Yeah, I think the only thing this test does is be an e2e version of `test_nan_is_read_as_string`. So I'd say let's remove it